### ISSUE TITLE
64 attached or detached BulletAppStates with parallel threading causes JVM to crash

### ISSUE THREAD

**Issue by James-Richardson** [jMonkeyEngine/jmonkeyengine#928]
If your game is routinely attaching and detaching `BulletAppState`s, once it attaches the 64th `BulletAppState`, the JVM crashes.

Note that the issue isn't that there are more than 64 `BulletAppState`s _currently_ attached. The JVM crashes if more than 64 `BulletAppState`s have previously _been_ attached, and even if the current number of attached `BulletAppState`s is far less than 64.

This issue can be reproduced with the following code, in which case only one `BulletAppState` is ever attached, but it is attached and detached more than 64 times:

```java
    private int count;
    private int frame;
    private BulletAppState bulletAppState;
    
    @Override
    public void simpleInitApp() {}
    
    @Override
    public void simpleUpdate(float tpf) {
        if (frame % 4 == 0) {
            System.out.println(++count);
            
            bulletAppState = new BulletAppState();
            
            bulletAppState.setThreadingType(ThreadingType.PARALLEL);
            
            stateManager.attach(bulletAppState);
        } else if (frame % 4 == 2) {
            stateManager.detach(bulletAppState);
        }
        
        frame++;
    }
```

More information on this issue can be found in the [original forum topic](https://hub.jmonkeyengine.org/t/solved-exception-access-violation-after-64-kinematicragdollcontrols/40952/22).

**Comment by stephengold** [comment]
Fixed in `master` branch.

**Comment by stephengold** [comment]
Note that the underlying issue was fixed in Bullet at https://github.com/bulletphysics/bullet3/commit/ee0e3ba2718cb636b178574c53d67fef2e6daaf7

That made disabling Bullet's Quickprof profiling completely unnecessary.

At some point, Bullet's mechanism for enabling/disabling profiling was changed. As a consequence, the fix for #928 actually doesn't disable Quickprof any more.
